package.json

```diff
@@ -37,23 +37,37 @@
   "license": "MIT",
   "analyze": true,
   "scripts": {
-    "test": "grunt all"
+    "test": "npm-run-all clean copy:json ts:* test:* lint:* validate:* copy:package copy:meta generate:*",
+    "start": "tsc --watch",
+    "clean": "rimraf dist",
+    "copy:json": "cpy \"src/**/*.json\" dist --parents",
+    "copy:package": "cpy \"**/*.js\" \"**/*.json\" \"!tests/**\" \"!references.js\" \"tests/TestHelper.js\" \"tests/TestHelper.d.ts\" ../build --cwd=\"dist/src\" --parents",
+    "copy:meta": "cpy README.md recommended_ruleset.js dist/build  --parents",
+    "ts:default": "tsc",
+    "ts:test-data": "tsc -p test-data",
+    "test:rules": "tslint -r dist/src --test \"tests/**\"",
+    "test:mocha": "mocha \"dist/src/tests/**/*.js\"",
+    "lint:rules": "tslint -r dist/src -t verbose -c tslint.json -e \"src/tests/**\" \"src/**/*.ts\"",
+    "lint:tests": "tslint -r dist/src -t verbose -c src/tests/tslint.json -e src/tests/references.ts \"src/tests/**/*.ts\"",
+    "validate:documentation": "node build-tasks/validate-documentation.js",
+    "validate:config": "node build-tasks/validate-config.js",
+    "generate:recommendations": "node build-tasks/generate-recommendations.js",
+    "generate:default-tslint-json": "node build-tasks/generate-default-tslint-json.js",
+    "generate:sdl-report": "node build-tasks/generate-sdl-report.js",
+    "generate:rule-metadata": "node build-tasks/generate-rule-metadata.js",
+    "generate:package-json-for-npm": "node build-tasks/generate-package-json-for-npm.js",
+    "create-rule": "node build-tasks/create-rule.js"
```

Comment:
Would you mind alphabetizing these rules? I always find it really difficult to scan through lists of tasks unless they're grouped nicely.

Reply:
It is possible to sort alphabetically by group/single tasks.
But current order inside "group" (e.g. `generate:<task>`) helps keep `test` script short enough.
As I've seen - `npm-run-all` preserves tasks order in case when invoked with as `npm-run-all generate:*` where order matters. Otherwise full script names should be specified in correct order.

Should I leave order inside group as per execution order (`create-rule` will be moved up, `test` & `start` down etc.) or should I reorder everything and explicitly set order inside `test` script?

Reply:
Leaving order inside the groups seems good, thanks!